apps/ewallet/lib/ewallet/web/v1/serializers/account_serializer.ex

```diff
@@ -31,6 +31,7 @@ defmodule EWallet.Web.V1.AccountSerializer do
       name: account.name,
       description: account.description,
       master: Account.master?(account),
+      category_ids: CategorySerializer.serialize(account.categories, :id),
```

Comment:
No `categories` field that can be preloaded to send back all the categories? For now, I don't see how we can get all the categories for an account 🤔